contrib/utilities/checkdoxygen.py

```diff
@@ -60,4 +91,26 @@ def check_empty_lines_in_tables(lines):
 check_doxygen_groups(lines)
 check_empty_lines_in_tables(lines)
 
+# if it's intro.dox, combine it with step-**.cc and results.dox and check headers
+if 'intro.dox' in args[0]:
+    # get stepname.cc
+    stepname = args[0].split('/doc',1)[0]
+    stepname = stepname + '/' + stepname.split('/',1)[1] + '.cc'
+
+    # get results.dox
+    resultsname = args[0].split('intro.dox',1)[0] + 'results.dox'
+    
+    # check the .cu file for step-64
+    if '64' in stepname:
```

Comment:
this is problematic, as this will break if we ever have other tutorials with cu. How about you unconditionally grab the .cc and the .cu (if they exist).

Reply:
I have a `try` `except` here if it can't find the `.cc`. The other option was to import another package that checks if files exist, but I imagine it just does a `try` `except` under the hood anyway.